packages/core/src/dialects/mssql/query-generator-internal.ts

```diff
@@ -45,4 +51,67 @@ export class MsSqlQueryGeneratorInternal<Dialect extends MssqlDialect = MssqlDia
 
     return fragment;
   }
+
+  getReturnFields(options: GetReturnFieldsOptions, modelDefinition?: ModelDefinition | null): string[] {
+    const returnFields: string[] = [];
+
+    if (Array.isArray(options.returning)) {
+      returnFields.push(...options.returning.map(field => {
```

Comment:
Comments made on the parent method apply here too

Reply:
Implementation update now.